src/entities/InductionClass.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity, PrimaryColumn, Column } from 'typeorm';
+
+@Entity()
+export class InductionClass {
+  @PrimaryColumn()
+  quarter: string;
+
+  @Column()
+  name: string;
+
+  @Column('date')
+  startDate: string;
```

Comment:
What does the start date and end date indicate here? I thought these induction classes are permanent?

Reply:
ah good call - i meant this to be induction period, so FA19 would be something like september 1st 2019 - whenever

Reply:
Oh okay, that's fine with me then.